web/src/transmission.js

```diff
@@ -45,6 +45,27 @@ export class Transmission extends EventTarget {
       this.action_manager.update(event_)
     );
 
+    // add drag and drop to body and 'open-torrent'
+    document.body.addEventListener('dragover', (event_) => {
+      event_.preventDefault();
+      // get upload-div div and set it as dropzone (to activate on dragover)
+      const upload_div = document.querySelector('.upload-div');
+      if (upload_div) {
```

Comment:
Since you are using ES6+ constructs anyway. You could use:
```javascript
upload_div ?? upload_div.classList.add('dropzone');
```
or even
```javascript
upload_div?.classList.add('dropzone');
```
But a few lines later you get the same dom-element again and add an event listener without any check at all.

Reply:
Ahh, upload_div is scoped to the above function, and hadn't even thought about moving it out. I'll do something similar!